compiler/next/include/chpl/uast/BlockStyle.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CHPL_UAST_BLOCKSTYLE_H
+#define CHPL_UAST_BLOCKSTYLE_H
+
+namespace chpl {
+namespace uast {
+
+/**
+  An enum representing whether a construct's body is preceded by a keyword,
+  a block, or both.
+*/
+enum struct BlockStyle {
+  IMPLICIT,
+  EXPLICIT,
+  UNNECESSARY_KEYWORD_AND_BLOCK
```

Comment:
👍 👍

Reply:
The two constants here are indirect and the third constant is way too long. While this probably doesn't matter, I would be much happier with:
```c++
enum struct BlockStyle {
  KEYWORD,
  BRACES,
  KEYWORD_AND_BRACES
};
```

Reply:
Some thoughts: The current enum is designed to (perhaps overly) emphasize the error case, for the sake of a prospective linter. If we were to change it to `KEYWORD_AND_BRACES`, then there would be some cases where that would not be an error (e.g. `do { ... } while ()`), though perhaps this is not an issue as we would lint on a node-by-node basis.

Personally I don't think `IMPLICIT` and `EXPLICIT` are that obtuse. But I don't have much stake in the game here and would be fine with either option.